compiler/rustc_ast/src/ast.rs

```diff
@@ -1553,7 +1563,8 @@ impl MacArgs {
         match self {
             MacArgs::Empty => None,
             MacArgs::Delimited(dspan, ..) => Some(dspan.entire()),
-            MacArgs::Eq(eq_span, token) => Some(eq_span.to(token.span)),
+            MacArgs::Eq(eq_span, MacArgsEqKind::Ast(expr)) => Some(eq_span.to(expr.span)),
+            MacArgs::Eq(eq_span, MacArgsEqKind::Hir(lit)) => Some(eq_span.to(lit.span)),
```

Comment:
Is this case reachable?

Reply:
It's not. I will mark it as unreachable.